websites/N/Neovel/dist/metadata.json

```diff
@@ -0,0 +1,45 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "RisingSunLight",
+		"id": "240521747852558347"
+	},
+	"service": "Neovel",
+	"description": {
+		"en": "Designed by and for book lovers ! Discover countless new stories at your fingertips and join a strong positive community !",
+		"fr": "Pensé et conçu pour les amoureux des novels ! Accédez, lisez et découvrez simplement des milliers de novels et rejoignez une communauté de passionnés !",
```

Comment:
```suggestion
		"fr": "Conçu par et pour les amoureux des livres ! Découvrez d'innombrables nouvelles histoires à portée de main et rejoignez une forte communauté positive !",
```

Reply:
Well, the description of the site in the metadata is not good then xD